057-AzureRouteServer/Student/Challenge-00.md

```diff
@@ -0,0 +1,76 @@
+# Challenge 00 - Prerequisites - Ready, Set, GO
+
+**[Home](../README.md)** - [Next Challenge >](./Challenge-01.md)
+
+## Introduction
+
+A smart Azure engineer always has the right tools in their toolbox. In addition, a good grasp on the key fundamental networking concepts. In this case, the Border Gateway Protocol (BGP).
+
+## Description
+
+In this challenge we'll be setting up all the tools we will need to complete our challenges.
+
+- Make sure that you have joined the Teams group for this track. Please ask your coach about the correct Teams channel to join.
+- Make sure you have an Azure Subscription. In case you don't, possibly ask your coach to create a Resource Group for you. 
```

Comment:
I don't think creating a resource group in the coach's Azure is a good idea (it's not clear in which Azure subscription you meant to create the resource group). If the attendees work for the same company, could they share subscription(s) or could they use an Azure trial instead?

Reply:
I have had students in other WTH where they don't have a sub and I end up creating a resource group on my subscription (coach) and giving them RBAC access. That's what I meant to say